dem/getting-started/real-user-monitoring.md

```diff
@@ -0,0 +1,29 @@
+---
+id: real-user-monitoring
+title: Real User Monitoring (RUM)
+---
+
+# Real User Monitoring (RUM)
+
+![image](../assets/getting-started/rum-1.png)
+
+Called “RUM”🍹 among insiders, Real User Monitoring consists of observing and analyzing the experience perceived by real users, directly from their browsers, whatever browser they use.
+
+It's important to note that:
+
+- this feature requires adding an **external tag** to the page, which is designed to be loaded **asynchronously** and to be **extremely lightweight** so it doesn't slow the user's browsing on the site.
+- the type of data sent by the browser via the Quanta tag and the way Quanta stores these elements in its database ensure **the Quanta tag is excluded from the scope of GDPR**. Indeed, the figures sent through the tag are purely technical and **not personally identifiable**. The Quanta dashboard allows observing site behavior for different browser types (Chrome, Mobile Safari, EDGE, ...) but without any possibility to identify a unique user.
```

Comment:
`for different browser types (Chrome, Mobile Safari, EDGE, ...)  but without any possibility to identify a unique user` -> `for different aggregated perspectives (e.g. the browser perspective will show metrics for Chrome, Edge, etc. across all recorded hits) without any possibility to identify a unique user.